coursebuilder/modules/explorer/_static/components/pages/homepage.html

```diff
@@ -186,15 +186,15 @@ <h2 class="section-label">Testimonials</h2>
       <template is="dom-if" if="[[!currentUser.loggedIn]]">
         <a href="[[currentUser.loginUrl]]">
           <div class="continue-learning">
-            <a class="waves-effect waves-light btn-large" target="_blank">Start Learning</a>
+            <a class="waves-effect waves-light btn-large" target="_blank" href="[[currentUser.loginUrl]]">Start Learning</a>
           </div>
         </a>
       </template>
 
       <template is="dom-if" if="[[currentUser.loggedIn]]">
         <a href="#!explorer">
           <div class="continue-learning">
-            <a class="waves-effect waves-light btn-large" target="_blank">Keep Learning</a>
+            <a class="waves-effect waves-light btn-large" target="_blank" href="#!explorer">Keep Learning</a>
```

Comment:
@Sharonsyra let's remove target="_blank" for this.

Reply:
@Sharonsyra Is there a reason why we are repeating the href from https://github.com/andela-angene/coursebuilder-core/blob/26175dba36e93d74695ea706f0abdd323b7f0004/coursebuilder/modules/explorer/_static/components/pages/homepage.html#L195

Reply:
Sorry, this was an oversight.

Reply:
@Sharonsyra it happens. "If debugging is the process of removing software bugs, then programming must be the process of putting them in." Lol!